pkg/errors/error.go

```diff
@@ -36,6 +36,8 @@ var (
 	ErrBitrotAlreadyEnabled    = errors.New("bitrot is already enabled")
 	ErrBitrotAlreadyDisabled   = errors.New("bitrot is already disabled")
 	ErrBitrotNotEnabled        = errors.New("bitrot is not enabled")
+	ErrProcessNotRunning       = errors.New("process is not running")
```

Comment:
Suggestion/question

instead of returning process is already running error.
can we be more specific while returning an error like which process is already running

Reply:
thinking of reuse, i had a common error. as we have a lot of errors that we can create as per need and also during restart situation, this change will be helpful to know which process isnt coming up.